page-changelog.php

```diff
@@ -15,6 +15,108 @@
 </ul>
 
 <a name="latest10.0"></a></li>
+<h3>Version 10.0.4 <small>Nov 22 2017</small><br></h3>
```

Comment:
since 10.0.4 isn't released yet I'd rather have this in another PR which we can merge once ready (and would also need adjusting release date)